### ISSUE TITLE
Question about libpd

### ISSUE THREAD

**Issue by bowd** [agraef/purr-data#9]
> Caveat: I'm very new to this universe so this might not make sense

I've just ordered a Bela board to try and bring to life this crazy loop-mixer thing in my head and I'm getting into pure-data but I'm a bit confused about to what extent purr-data is a drop-in replacement for pure-data. So, can I use `libpd` with purr-data and upload the patches to by Bela dev-kit?

**Comment by agraef** [comment]
Nope, for libpd support you still need vanilla. But patches created with purr data should probably work with libpd, as long as you only use the vanilla features (better test them beforehand with vanilla, though).
